storage/rocksdb/rdb_cf_options.h

```diff
@@ -67,7 +68,8 @@ class Rdb_cf_options {
   std::shared_ptr<rocksdb::MergeOperator>
   get_cf_merge_operator(const std::string &cf_name);
 
-  void get_cf_options(const std::string &cf_name,
+  /* bool true return indicates cf_name was found, otherwise default */
+  bool get_cf_options(const std::string &cf_name,
```

Comment:
Likewise

Reply:
Actually, there is one location that calls get_cf_options that does not care about the return code, that is from the plugin init func at ha_rocksdb.cc:4150, so that can't work on this func.